src/gui/OptionsMenu.cc

```diff
@@ -494,7 +494,7 @@ class VideoCheckTickDown : public Label {
         int tilesize = vts->tilesize;
         std::stringstream ss;
         if (value == 0)
-            ss << "Not fixed";
+            ss << N_("Not fixed");
```

Comment:
Changed line: localization marker.

That's all she wrote, the rest is trailing whitespace.